src/components/SearchBar/index.tsx

```diff
@@ -0,0 +1,15 @@
+import S from './Searchbar.styled';
+
+import type { SearchBarProps } from './types';
+
+const SearchBar = ({ placeholder, isWrite = false, onClick, ...args }: SearchBarProps) => {
+  return (
+    <S.SearchBar>
+      <S.SearchBarInput placeholder={placeholder} {...args} />
+      <S.DeleteIcon isWrite={isWrite} onClick={onClick} />
+      <S.SearchIcon onClick={onClick} />
```

Comment:
```suggestion
      <S.DeleteIcon isWrite={isWrite} onClick={onClickDelete} />
      <S.SearchIcon onClick={onClickSearch} />
```
온클릭함수는 별도로 받아야하지 않을까요??

Reply:
@haesoo-y
넵 수정 완료하였습니다!